src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
```

Comment:
```suggestion
```

Reply:
so why checkstyle plugin don`t see this mistake?